.github/workflows/gateway-conformance.yml

```diff
@@ -0,0 +1,146 @@
+name: Gateway Conformance Dashboard
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  # Enable this when we're ready to generate the dashboard
+  # schedule:
+  #   - cron: "0 */6 * * *" # every six hours
+
+concurrency:
+  group: "conformance"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  configure:
+    runs-on: ubuntu-latest
+    outputs:
+      gateways: ${{ steps.set-matrix.outputs.gateways }}
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+     - name: install yq
+       env:
+          VERSION: v4.35.1
+          BINARY: yq_linux_amd64
+       run: |
+        wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/local/bin/yq
+        chmod +x /usr/local/bin/yq
```

Comment:
This one has a different syntax than jq, right? Generally, I prefer https://github.com/kislyuk/yq myself, but nothing wrong with this one either.

However, are you downloading it to make sure the version is pinned? If not, then I think there should be some version of that yq already installed.

Reply:
(dropping yq to use raw txt files)